src/txmempool.h

```diff
@@ -623,7 +629,10 @@ class CTxMemPool
     /** Affect CreateNewBlock prioritisation of transactions */
     void PrioritiseTransaction(const uint256& hash, const CAmount& nFeeDelta);
     void ApplyDelta(const uint256& hash, CAmount &nFeeDelta) const EXCLUSIVE_LOCKS_REQUIRED(cs);
-    void ClearPrioritisation(const uint256& hash) EXCLUSIVE_LOCKS_REQUIRED(cs);
+    void ClearPrioritisation(const uint256& hash);
```

Comment:
why no longer EXCLUSIVE_LOCKS_REQUIRED ?

Reply:
Good catch, that was a mistake. Restored in: https://github.com/SequentiaSEQ/SEQ-Core-Elements/pull/10/commits/bbd655ff33df74f279605fad764109916b8dabbe